packages/api/install.go

```diff
@@ -54,149 +56,115 @@ type installParams struct {
 }
 
 func installCommon(data *installParams, logger *log.Entry) (err error) {
-	if IsInstalled() || model.DBConn != nil || config.IsExist() {
+
+	if conf.Installed {
 		return fmt.Errorf(`E_INSTALLED`)
 	}
-	if data.generateFirstBlock {
-		*utils.GenerateFirstBlock = 1
-	}
+
 	if data.logLevel != "DEBUG" {
```

Comment:
I think this is unnecessary strings. Let the user choose any log level

Reply:
Removed.